src/components/Proposal/ProposalContainer.tsx

```diff
@@ -114,12 +114,18 @@ class ProposalContainer extends React.Component<IProps, null> {
         rewards.push(proposal.nativeTokenReward + " " + dao.tokenSymbol);
       }
       if (proposal.reputationChange) {
-        rewards.push(proposal.reputationChange + " reputation");
+        rewards.push(
+          <Tooltip placement="top" trigger={["hover"]} overlay={<span style={{minHeight: 'inherit !important'}}>{proposal.reputationChange} reputation</span>}>
+            <span style={{cursor: 'pointer'}}>
```

Comment:
a pointer usually means you can click on the element and have it do something, I dont think we want that here

Reply:
Idk, Feels more natural to me, do you have another suggestion? Or shall I remove it?

Reply:
I would just remove it. the tooltip appears immediately anyway right?

Reply:
Yes